api/formatters/covjson.py

```diff
@@ -27,7 +27,9 @@
 
 def make_parameter(ts_mdata):
     return Parameter(
-        description={"en": ts_mdata.title},
+        description={
+            "en": "For detailed description see observedProperty id"
+        },  # ts_mdata.title}, # title is unique for each TS, and does not describe a parameter
```

Comment:
The description parameter is not mandatory. I think it is better to just drop it then have this hardcoded string.

But just referring to the observedProperty id is not good enough, as that describes for example air temperature, where there should be someplace to give more detailed info.

At KNMI we have a parameter level description (e.g. "Ambient Temperature 1.5m 10 Min Minimum") which is useful to have here for the user. Could we somehow have metadata which we say has to be at parameter level? Or maybe it could be constructed from other fields.

Can you give an example of how this comes out unique for each TS at Met Norway?